cli/testdata/streamlit_app/.gitignore

```diff
@@ -0,0 +1 @@
+.tool_id.txt
```

Comment:
Why this?

Reply:
I think it is there because the folder came from extraction of an older packaged app, created with `init` :) We do not need the `.gitignore` here, so I'll just remove it.